doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -17,7 +17,7 @@ command:
 include::../protos/vkGetInstanceProcAddr.txt[]
 
   * pname:instance is the instance that the function pointer will be
-    compatible with.
+    compatible with, or `NULL` for commands not dependent on any instance.
```

Comment:
That should be everywhere for handle types...
It is used correctly on some other commands (in VU section). There must be some bug in the script translating the `optional` `vk.xml` attribute, ending up  `NULL` instead for this command...